### PR TITLE
add v6.1.2 release notes

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.1.2.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#5010](https://github.com/pingcap/tiflow/issues/5010)
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - DM precheck no longer reports lacking privileges of INFORMATION_SCHEMA [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - (dup) 修复 DM 报错 `Specified key was too long` 的问题 [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) 修复数据同步过程中，latin1 字符集数据可能损坏的问题 [#7028](https://github.com/pingcap/tiflow/issues/7028)
+
+    + TiCDC **owner: @nongfushanquan**
+
+        - Fix a bug that may cause cdc server panic if it received a http request before cdc server fully started [#6838](https://github.com/pingcap/tiflow/issues/6838)
```
Comment (nongfushanquan) [inline releases/release-6.1.2.md:117]: - 修复了 cdc server 在没启动成功前收到 http 请求导致 panic 的问题  [#6838](https://github.com/pingcap/tiflow/issues/6838)
         - 修复了日志太多的问题 [#7235](https://github.com/pingcap/tiflow/issues/7235)
         - 修复 redo log 中错误清理非当前 changefeed 日志文件的问题 [#6413](https://github.com/pingcap/tiflow/issues/6413)
         - 修复在一个 etcd 事务中提交太多数据导致 cdc 服务不可用问题  [#7131](https://github.com/pingcap/tiflow/issues/7131)
        -修复Redo log 中 DDL 重复执行可能导致的数据不一致性问题  [#6927]https://github.com/pingcap/tiflow/issues/6927
- 采用批处理 resolved ts 的模式，提升 region worker 的性能[#7078]https://github.com/pingcap/tiflow/issues/7078

File: releases/release-6.1.2.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        - Change log level from info to debug for some logs to avoid too many logs [#7235](https://github.com/pingcap/tiflow/issues/7235)
+        - Fix a bug that may cause changefeed's redo log files be deleted wrongly [#6413](https://github.com/pingcap/tiflow/issues/6413)
+        - Fix a bug that may cause cdc unavaliable by commit too many operation in a etcd transaction [#7131](https://github.com/pingcap/tiflow/issues/7131)
+        - Fix a bug which can lead inconsistency Change if non-reentrant DDLs can be executed twice [#6927](https://github.com/pingcap/tiflow/issues/6927)
+        - Enhance the region worker's performance by handling the resolved ts in the batch mode [#7078](https://github.com/pingcap/tiflow/issues/7078)
```
Comment (Oreoxmt) [inline releases/release-6.1.2.md:122]:
```suggestion
        - 修复了 cdc server 在没启动成功前收到 HTTP 请求导致 panic 的问题  [#6838](https://github.com/pingcap/tiflow/issues/6838)
        - 修复了日志太多的问题 [#7235](https://github.com/pingcap/tiflow/issues/7235)
        - 修复 redo log 中错误清理非当前 changefeed 日志文件的问题 [#6413](https://github.com/pingcap/tiflow/issues/6413)
        - 修复在一个 etcd 事务中提交太多数据导致 cdc 服务不可用问题  [#7131](https://github.com/pingcap/tiflow/issues/7131)
        - 修复 Redo log 中 DDL 重复执行可能导致的数据不一致性问题  [#6927](https://github.com/pingcap/tiflow/issues/6927)
        - 采用批处理 resolved ts 的模式，提升 region worker 的性能[#7078](https://github.com/pingcap/tiflow/issues/7078)
```

File: releases/release-6.1.2.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    - (dup) 修复 `SHOW CREATE PLACEMENT POLICY` 输出结果不正确的问题 [#37526](https://github.com/pingcap/tidb/issues/37526)
+    - (dup) 修复了当某台 PD 宕机时，由于没有重试其他 PD 节点，导致查询表 `INFORMATION_SCHEMA.TIKV_REGION_STATUS` 时请求失败的问题 [#35708](https://github.com/pingcap/tidb/issues/35708)
+    - (dup) 修复 `UNION` 运算符可能会非预期地返回空结果的问题 [#36903](https://github.com/pingcap/tidb/issues/36903)
+
+    <!--execution **owner: @zanmato1984**-->
+
+    - Database level privileges are cleaned up correctly [#38363](https://github.com/pingcap/tidb/issues/38363)
+
+    <!--transaction **owner: @cfzjywxk**-->
+
```
Comment (cfzjywxk) [inline releases/release-6.1.2.md:71]: - 修复 region 合并情况下 region cache 没有及时清理的问题 [#37141](https://github.com/pingcap/tidb/issues/37141)
- 修复 kv client 发送不必要 ping 消息问题 [#36861](https://github.com/pingcap/tidb/issues/36861)

Comment (Oreoxmt) [reply releases/release-6.1.2.md:69]: @cfzjywxk Add in https://github.com/pingcap/docs-cn/pull/11678/commits/7a276ba42fd8703c07a3b8b0eb0d1f7184c53742 and https://github.com/pingcap/docs/pull/10883/commits/0dc084eded5f28a141ee73884d21ffee7b8ad85d

**Review by fixdb — APPROVED** [review]
+1

File: releases/release-6.1.2.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    - Fix the issue that the statistics of the region tree may be not accurate [#5318](https://github.com/tikv/pd/issues/5318)
+    - (dup) 修复 PD 可能没创建 TiFlash Learner 副本的问题 [#5401](https://github.com/tikv/pd/issues/5401)
+    - (dup) 修复 PD 无法正确处理 dashboard 代理请求的问题 [#5321](https://github.com/tikv/pd/issues/5321)
+    - (dup) 修复不健康的 Region 可能导致 PD panic 的问题 [#5491](https://github.com/tikv/pd/issues/5491)
+
++ TiFlash
+
+    <!--compute **owner: @zanmato1984**-->
+
+    - (dup) 修复取消查询时 window function 可能会导致 TiFlash 崩溃的问题 [#5814](https://github.com/pingcap/tiflash/issues/5814)
```
Comment (flowbehappy) [inline releases/release-6.1.2.md:97]:
```suggestion
    - (dup) 修复取消查询时 window function 可能会导致 TiFlash 崩溃的问题 [#5814](https://github.com/pingcap/tiflash/issues/5814)
    - 修复在大批量写入之后，iolimiter 可能错误的限制了读请求的 IO 吞吐量，从而降低查询性能的问题 [#5801](https://github.com/pingcap/tiflash/issues/5801)
```

**Review by 3pointer — APPROVED** [review]

File: releases/release-6.1.2.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    - Fix the issue that GROUP CONCAT with ORDER BY might fail when the ORDER BY clause contains a correlated subquery [#18216](https://github.com/pingcap/tidb/issues/18216)
+    - (dup) 修复 `UPDATE` 语句中带公共表表达式 (CTE) 的情况下会报 `Can't find column` 的问题 [#35758](https://github.com/pingcap/tidb/issues/35758)
+    - (dup) 修复某些情况下，`EXECUTE` 语句可能抛出非预期异常的问题 [#37187](https://github.com/pingcap/tidb/issues/37187)
+
++ TiKV **owner: @ethercflow**
+
+    - (dup) 修复开启流量控制且显式设置 `level0_slowdown_trigger` 时出现 QPS 下降的问题 [#11424](https://github.com/tikv/tikv/issues/11424)
+    - (dup) 修复 Web 身份提供程序 (web identity provider) 报错并失效后，自动恢复为默认提供程序 (default provider) 时出现权限拒绝的问题 [#13122](https://github.com/tikv/tikv/issues/13122)
+    - (dup) 修复当有一个 TiKV 实例出现网络隔离时，一段时间内服务不可用问题 [#12966](https://github.com/tikv/tikv/issues/12966)
+
```
Comment (ethercflow) [inline releases/release-6.1.2.md:85]:
```suggestion
- 修复因引入跨 region 批量 snapshot 导致 snapshot 数据不完整的问题 [#13553](https://github.com/tikv/tikv/issues/13553)

```

File: releases/release-6.1.2.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+## 兼容性变更
+
+## 提升改进
+
++ TiDB
+
+    <!--sql-infra **owner: @wjhuang2016**-->
+
+    - (dup) 解决基于 SQL 的数据放置规则功能和构建 TiFlash 副本功能的冲突 [#37171](https://github.com/pingcap/tidb/issues/37171)
```
Comment (wjhuang2016) [inline releases/release-6.1.2.md:19]: 这个有点难理解。
原来是：允许在一张表上同时设置数据放置规则和TiFlash 副本数。

Comment (Oreoxmt) [reply releases/release-6.1.2.md]: @wjhuang2016 已修改 https://github.com/pingcap/docs-cn/pull/11678/commits/ec1e8000b84174eac83d28b8300e4215d6ca3bea

**Review by ethercflow — APPROVED** [review]
LGTM

File: releases/release-6.1.2.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
++ TiFlash
+
+<!--compute **owner: @zanmato1984**-->
+
+<!--storage **owner: @flowbehappy**-->
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
```
Comment (niubell) [inline releases/release-6.1.2.md:42]: Checksum 增加更多 TiKV 重试错误，提升成功率 [#37690](https://github.com/pingcap/tidb/issues/37690)

File: releases/release-6.1.2.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+    - (dup) 修复取消查询时 window function 可能会导致 TiFlash 崩溃的问题 [#5814](https://github.com/pingcap/tiflash/issues/5814)
+
+    <!--storage **owner: @flowbehappy**-->
+
+    - (dup) 修复使用包含 `NULL` 值的列创建主键时导致崩溃的问题 [#5859](https://github.com/pingcap/tiflash/issues/5859)
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
+
```
Comment (niubell) [inline releases/release-6.1.2.md:107]: 修复 Metrics 打点可能导致 Panic 的问题 [#37338](https://github.com/pingcap/tidb/issues/37338)

File: releases/release-6.1.2.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+    + TiDB Data Migration (DM) **owner: @niubell**
+
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#5010](https://github.com/pingcap/tiflow/issues/5010)
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - DM precheck no longer reports lacking privileges of INFORMATION_SCHEMA [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - (dup) 修复 DM 报错 `Specified key was too long` 的问题 [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) 修复数据同步过程中，latin1 字符集数据可能损坏的问题 [#7028](https://github.com/pingcap/tiflow/issues/7028)
+
```
Comment (niubell) [inline releases/release-6.1.2.md:115]: 修复开启 Validator 时 DM Workers 可能触发死锁问题 [#7241](https://github.com/pingcap/tiflow/issues/7241)

File: releases/release-6.1.2.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
+
+        - 修复 Metrics 打点可能导致 Panic 的问题 [#37338](https://github.com/pingcap/tidb/issues/37338)
+
+    + TiDB Data Migration (DM) **owner: @niubell**
+
+        - DM will try to persist upstream table structure from dump files when firstly switch to sync unit [#5010](https://github.com/pingcap/tiflow/issues/5010)，[#7159](https://github.com/pingcap/tiflow/issues/7159)
```
Comment (niubell) [inline releases/release-6.1.2.md:114]: DM 在第一次切换到 Sync 阶段时持久化上游表结构解决 Sync 可能失败问题，同时拆分 SQL 解决大事务问题 [#5010](https://github.com/pingcap/tiflow/issues/5010)，[#7159](https://github.com/pingcap/tiflow/issues/7159)
解决 Pre-Check 阶段对 INFORMATION_SCHEMA 表需要 SELECT 权限问题 [#7317](https://github.com/pingcap/tiflow/issues/7317)

**Review by nolouch — APPROVED** [review]

File: releases/release-6.1.2.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+        - (dup) 修复 DM 报错 `Specified key was too long` 的问题 [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) 修复数据同步过程中，latin1 字符集数据可能损坏的问题 [#7028](https://github.com/pingcap/tiflow/issues/7028)
+
+    + TiCDC **owner: @nongfushanquan TW: @hfxsd**
+
+        - 修复了 cdc server 在没启动成功前收到 HTTP 请求导致 panic 的问题 [#6838](https://github.com/pingcap/tiflow/issues/6838)
+        - 修复了日志太多的问题 [#7235](https://github.com/pingcap/tiflow/issues/7235)
+        - 修复 redo log 中错误清理非当前 changefeed 日志文件的问题 [#6413](https://github.com/pingcap/tiflow/issues/6413)
+        - 修复在一个 etcd 事务中提交太多数据导致 cdc 服务不可用问题 [#7131](https://github.com/pingcap/tiflow/issues/7131)
+        - 修复 Redo log 中 DDL 重复执行可能导致的数据不一致性问题 [#6927](https://github.com/pingcap/tiflow/issues/6927)
```
Comment (hfxsd) [inline releases/release-6.1.2.md:126]:
```suggestion
        - 修复了 cdc server 在尚未启动成功前收到 HTTP 请求导致 panic 的问题 [#6838](https://github.com/pingcap/tiflow/issues/6838)
        - 修复了日志数量过多的问题 [#7235](https://github.com/pingcap/tiflow/issues/7235)
        - 修复 redo log 中错误清理非当前 changefeed 日志文件的问题 [#6413](https://github.com/pingcap/tiflow/issues/6413)
        - 修复在一个 etcd 事务中提交太多数据导致 cdc 服务不可用问题 [#7131](https://github.com/pingcap/tiflow/issues/7131)
        - 修复 redo log 中 DDL 重复执行可能导致的数据不一致性问题 [#6927](https://github.com/pingcap/tiflow/issues/6927)
```

File: releases/release-6.1.2.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+        - 采用批处理 resolved ts 的模式，提升 region worker 的性能 [#7078](https://github.com/pingcap/tiflow/issues/7078)
```
Comment (hfxsd) [inline releases/release-6.1.2.md:127]: 这个看起来不像 bug，是否应该挪到 提升改进 下面？ @leoppro

File: releases/release-6.1.2.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+    - (dup) 修复了当某台 PD 宕机时，由于没有重试其他 PD 节点，导致查询表 `INFORMATION_SCHEMA.TIKV_REGION_STATUS` 时请求失败的问题 [#35708](https://github.com/pingcap/tidb/issues/35708)
+    - (dup) 修复 `UNION` 运算符可能会非预期地返回空结果的问题 [#36903](https://github.com/pingcap/tidb/issues/36903)
+
+    <!--execution **owner: @zanmato1984**-->
+
+    - (dup) 修复在 TiFlash 中为分区表开启动态模式时结果出错的问题 [#37254](https://github.com/pingcap/tidb/issues/37254)
+
+    <!--transaction **owner: @cfzjywxk**-->
+
+    - 修复 Region 合并情况下 Region cache 没有及时清理的问题 [#37141](https://github.com/pingcap/tidb/issues/37141)
```
Comment (TomShawn) [inline releases/release-6.1.2.md:71]:
```suggestion
    - 修复 Region 合并情况下 Region cache 没有及时被清理的问题 [#37141](https://github.com/pingcap/tidb/issues/37141)
```

File: releases/release-6.1.2.md [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+    - 修复 KV client 发送不必要 ping 消息的问题 [#36861](https://github.com/pingcap/tidb/issues/36861)
```
Comment (TomShawn) [inline releases/release-6.1.2.md:72]:
```suggestion
    - 修复 KV 客户端发送不必要 ping 消息的问题 [#36861](https://github.com/pingcap/tidb/issues/36861)
```

File: releases/release-6.1.2.md [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+    - 修复 Region tree 统计可能不准确的问题 [#5318](https://github.com/tikv/pd/issues/5318)
+    - (dup) 修复 PD 可能没创建 TiFlash Learner 副本的问题 [#5401](https://github.com/tikv/pd/issues/5401)
+    - (dup) 修复 PD 无法正确处理 dashboard 代理请求的问题 [#5321](https://github.com/tikv/pd/issues/5321)
+    - (dup) 修复不健康的 Region 可能导致 PD panic 的问题 [#5491](https://github.com/tikv/pd/issues/5491)
+
++ TiFlash **TW: @Oreoxmt**
+
+    <!--compute **owner: @zanmato1984**-->
+
+    - 修复在大批量写入之后，I/O Limiter 可能错误地限制了读请求的 IO 吞吐量，从而降低查询性能的问题 [#5801](https://github.com/pingcap/tiflash/issues/5801)
```
Comment (TomShawn) [inline releases/release-6.1.2.md:99]:
```suggestion
    - 修复在大批量写入之后，I/O Limiter 可能错误地限制了读请求的 I/O 吞吐量，从而降低查询性能的问题 [#5801](https://github.com/pingcap/tiflash/issues/5801)
```

File: releases/release-6.1.2.md [hunk 16]
```diff
@@ -0,0 +1,134 @@
...
+
+    + TiDB Lightning **owner: @niubell TW: @shichun-0415**
+
+        - 修复 Metrics 打点可能导致 Panic 的问题 [#37338](https://github.com/pingcap/tidb/issues/37338)
+
+    + TiDB Data Migration (DM) **owner: @niubell TW: @shichun-0415**
+
+        - DM 在第一次切换到 Sync 阶段时持久化上游表结构解决 Sync 可能失败问题，同时拆分 SQL 解决大事务问题 [#5010](https://github.com/pingcap/tiflow/issues/5010)，[#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - 解决 Pre-Check 阶段对 INFORMATION_SCHEMA 表需要 SELECT 权限问题 [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - 修复开启 Validator 时 DM Workers 可能触发死锁问题 [#7241](https://github.com/pingcap/tiflow/issues/7241)
```
Comment (shichun-0415) [inline releases/release-6.1.2.md:116]:
```suggestion
        - 修复某些情况下，进入 sync unit 的任务中断导致的上游表结构信息丢失问题 [#7159](https://github.com/pingcap/tiflow/issues/7159)
        - 通过在保存 checkpoint 时拆分 SQL 语句解决大事务问题 [#5010](https://github.com/pingcap/tiflow/issues/5010)
        - 解决 Pre-Check 阶段 `INFORMATION_SCHEMA` 表需要 `SELECT` 权限的问题 [#7317](https://github.com/pingcap/tiflow/issues/7317)
        - 修复开启 Validator 时 DM Workers 可能触发死锁问题 [#7241](https://github.com/pingcap/tiflow/issues/7241)
```

File: releases/release-6.1.2.md [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+
+<!--compute **owner: @zanmato1984**-->
+
+<!--storage **owner: @flowbehappy**-->
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell TW: @shichun-0415**
+
+        - 增加 checksum 阶段可重试错误，提升鲁棒性 [#37690](https://github.com/pingcap/tidb/issues/37690)
```
Comment (hfxsd) [inline releases/release-6.1.2.md:44]:
```suggestion
        - 增加 checksum 阶段可重试错误 (retryable error)，提升鲁棒性 [#37690](https://github.com/pingcap/tidb/issues/37690)
```

File: releases/release-6.1.2.md [hunk 18]
```diff
@@ -0,0 +1,135 @@
...
+    + TiDB Lightning **owner: @niubell TW: @shichun-0415**
+
+        - 修复非法 metric 数值可能导致 TiDB Lightning panic 的问题 [#37338](https://github.com/pingcap/tidb/issues/37338)
+
+    + TiDB Data Migration (DM) **owner: @niubell TW: @shichun-0415**
+
+        - 修复某些情况下，进入 sync unit 的任务中断导致的上游表结构信息丢失问题 [#7159](https://github.com/pingcap/tiflow/issues/7159)
+        - 通过在保存 checkpoint 时拆分 SQL 语句解决大事务问题 [#5010](https://github.com/pingcap/tiflow/issues/5010)
+        - 解决 Pre-Check 阶段 `INFORMATION_SCHEMA` 表需要 `SELECT` 权限的问题 [#7317](https://github.com/pingcap/tiflow/issues/7317)
+        - 修复开启 fast/full validator 时 DM Workers 可能触发死锁问题 [#7241](https://github.com/pingcap/tiflow/issues/7241)
```
Comment (hfxsd) [inline releases/release-6.1.2.md:117]:
```suggestion
        - 修复开启 fast/full validator 时 DM-worker 可能触发死锁问题 [#7241](https://github.com/pingcap/tiflow/issues/7241)
```

**Review by flowbehappy — APPROVED** [review]

**Review by XuHuaiyu — APPROVED** [review]

**Review by niubell — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by wjhuang2016 — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**